src/main/java/com/alkemy/ong/controller/UserController.java

```diff
@@ -19,4 +19,15 @@
         this.userService.deleteUserById(id);
         return ResponseEntity.status(HttpStatus.NO_CONTENT).build();
     }
+    @PatchMapping("/user/{id}")
+    public ResponseEntity<UserDTO> update( @PathVariable String id) {
+        Optional<UserEntity> entities = userRepository.findById(id);
+        if (entities.isPresent()) {
```

Comment:
I believe this validation should be in UserService implementation, is there any change in the UserService interface?

Reply:
change done